components/atom/backToTop/src/index.js

```diff
@@ -16,95 +16,83 @@ const STYLES = {
   LIGHT: 'light'
 }
 
-class AtomBackToTop extends PureComponent {
-  state = {
-    show: null,
-    hover: false
-  }
-
-  intervalId = 0
-  container = null
-
-  scrollStep = () => {
-    const {scrollTop} = this.container
-    const {scrollSteps} = this.props
-    const {intervalId} = this
+const AtomBackToTop = ({
+  iconTop: IconTop,
+  textTop,
+  scrollSteps,
+  scrollIntervalTime,
+  refContainer,
+  style
+}) => {
+  const [show, setShow] = useState(null)
+  const [hover, setHover] = useState(false)
+
+  let intervalId = 0
+  let container = useRef(null)
+
+  container = getTarget(refContainer)
```

Comment:
This could break SSR, getting the target and the properties below should be done in an effect.

Reply:
Moved getting the container element inside the effect 👍